models/load_redeemer.py

```diff
@@ -9,6 +9,17 @@
 from models.load_utils import to_markdown
 
 
+def remove_strings(page_content: str) -> str:
+    """This function removed unwanted strings."""
+    # List of strings to remove
+    strings_to_remove = ["|", "-"]
```

Comment:
it seems like sometimes a - is a valid character. Like 1830-1846, if you removed the - it would become 18301846, which would be hard to read. Maybe we need to leave - in there?